README.md

```diff
@@ -0,0 +1,19 @@
+# Agoric Oracle Middleware
+
+## Create oracles.json file
+
+In order to monitor oracles, you need to create a file at <b>~/config/oracles.json</b>. 
+As can be seen below, it takes an array of oracles with their addresses and names.
+
+```json
+{
+  "agoric12345aaaaaaaaaa" : { "oracleName": "Oracle 1" },
+  "agoric678910bbbbbbbbb" : { "oracleName": "Oracle 2" }
+}
+```
+
+## To run both the middleware and monitoring script
+
+```bash
+docker-compose up -d
+```
```

Comment:
Add .DS_Store to .gitignore

Reply:
Fixed in https://github.com/jacquesvcritien/agoric-cl-middleware/commit/707949bf7707bc469e119e34cc1a56479443a43b